IdentityCore/src/util/NSURL+MSIDBrokerRedirectUri.m

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.  
+
+
+#import "NSURL+MSIDBrokerRedirectUri.h"
+
+@implementation NSURL (MSIDBrokerRedirectUri)
+
++ (NSURL *)defaultNonBrokerRedirectUri:(NSString *)clientId
+{
+    if ([NSString msidIsStringNilOrBlank:clientId])
+    {
+        return nil;
+    }
+    
+    NSString *redirectUri = [NSString stringWithFormat:@"msal%@://auth", clientId];
```

Comment:
Can we check telemetry to see if this one is still used by anyone? Maybe it is a good time to drop support for this one :)

Reply:
Would you provide more guidance on how to check this one? I didn't see there is telemetry around the caller.

Reply:
I believe you can look it up from ESTS telemetry (e.g. check MaskedResponse field)